tests/cases/fourslash/renameStringLiteralTypes.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+
+////interface AnimationOptions {
+////    deltaX: number;
+////    deltaY: number;
+////    easing: "ease-in" | "ease-out" | "[|ease-in-out|]";

```

Comment:
Would this work if you tried renaming adjacent to the dashed portions? Would it work if you had spaces around?